modules/core/src/test/scala/org/tessellation/domain/statechannel/StateChannelValidatorSuite.scala

```diff
@@ -210,6 +229,25 @@ object StateChannelValidatorSuite extends MutableIOSuite {
       )
   }
 
+  test("should succeed when no signature is on the state channel allowance list for given address but is validated as historical") { res =>
```

Comment:
This test and...

Reply:
https://github.com/Constellation-Labs/tessellation/pull/698#discussion_r1274798615

Reply:
fixed :+1: